src/desktop/file_chooser.rs

```diff
@@ -522,22 +516,18 @@ impl SaveFileRequest {
     /// Sets the current folder.
     #[must_use]
     pub fn current_folder<P: AsRef<Path>>(mut self, current_folder: impl Into<Option<P>>) -> Self {
```

Comment:
Could those use `TryInto<FileName>` instead?

Reply:
I think not, we cannot implement `impl<S: AsRef<Path>> TryFrom<S> for FileName` since it conflicts with others.